auth/server/src/main/java/com/seagates3/response/formatter/AbstractResponseFormatter.java

```diff
@@ -64,4 +64,10 @@ abstract ServerResponse formatGetResponse(
 
  public
   abstract ServerResponse formatChangePasswordResponse(String operation);
+
+ public
+  abstract ServerResponse formatAttachUserPolicyResponse(String operation);
```

Comment:
This is not required as AttachUserPolicy and DetachUserPolicy doesnt return any response back.